src/server/worker/stats/stats_test.go

```diff
@@ -24,9 +24,9 @@ import (
 	prom_model "github.com/prometheus/common/model"
 )
 
-func activateEnterprise(c *client.APIClient) error {
```

Comment:
Nit: this looks basically identical to the new code in `src/server/auth/server/testing/admin_test.go` - it might be worth moving this into a common area and reusing it.